src/client/automation/playback/click/index.js

```diff
@@ -284,7 +280,9 @@ export default class ClickAutomation {
         var moveArguments = this._getMoveArguments();
 
         return this._move(moveArguments)
-            .then(() => this._mousedown())
+        // NOTE: we should raise mouseup event with ACTION_STEP_DELAY after we trigger
+        // mousedown event regardless of how long mousedown event handlers were executing
```

Comment:
yes, but it's auto-formatting. Maybe it's better to place the comment before `return this._move(moveArguments)`